### PR TITLE
NEW : Reopen Expense Report

### PR DESCRIPTION
**Pull request by zahwyrix**
# NEW
Issue #29787

### CODE REVIEW

File: htdocs/expensereport/card.php [hunk 1]
```diff
@@ -1545,6 +1552,10 @@
 
 	print '</form>';
 } elseif ($id > 0 || $ref) {
+	if ($action == 'reopen') {
+		$formconfirm = $form->formconfirm($_SERVER["PHP_SELF"].'?id='.$object->id, $langs->trans('ReOpen'), $langs->trans('ConfirmReOpenExpRep', $object->ref), 'confirm_reopen', '', 0, 1);
```
Comment (eldy) [inline htdocs/expensereport/card.php:1556]: key "ConfirmReOpenExpRep" must be added into en_US/trips.lang file.

**Comment by eldy** [comment]
In the workflow described into page:
https://wiki.dolibarr.org/index.php?title=Module_Expense_Reports

From which letter to which letter is the new flow ?

**Comment by zahwyrix** [comment]
You need to add an arrow from D to B, it's to be able to cancel a rejection, which returns the expense report to validated status.

**Comment by JonBendtsen** [comment]
@zephyriony @eldy what is the status on this PR? did it get forgotten?

**Pull request closed without merging**